suites/pacific/upgrades/multihop_upgrade_to_5.x.yaml

```diff
@@ -0,0 +1,77 @@
+# Upgrade: Multihop upgrade evaluation
+#
+# This test suite is used to test upgrades from all previous ceph release versions
+# to the latest 5.x version. We start from RHCS 4 and install a CDN cluster
+# then proceed with upgrading to every released version after that
+# finally upgrading the cluster to the latest 5.x version selected from the CLI
+# using the configuration given in the suite
+#
+
+# RHCS 5.x multihop upgrade test suite.
+tests:
+- test:
+    name: install ceph pre-requisites
+    module: install_prereq.py
+    abort-on-fail: True
+
+- test:
+    name: upgrade the RHCS cluster based on the path specified
+    polarion-id: ""
+    module: test_upgrade.py
+    config:
+      paths:
+        4.2:
```

Comment:
We can still specify cluster config for both the clusters in case of multisite by specifying the config under key "4.2" as given in this suite with different cluster names right? https://github.com/red-hat-storage/cephci/blob/master/suites/nautilus/rgw/sanity_rgw_multisite.yaml#L11-L103